code/modules/reagents/reagent_containers/glass_containers.dm

```diff
@@ -225,7 +227,7 @@
 
 /obj/item/reagent_containers/glass/beaker/vial
 	name = "vial"
-	desc = "A small glass vial. Can hold up to 25 units."
+	desc = "A small glass vial."
```

Comment:
A little worldbuilding could be good. These vials can only be found in the ussp ruin. In game, we're in the 26th century, so this would give them a more dated vibe.
```suggestion
	desc = "A small glass vial, often used by virologists of the 25th century."
```

Reply:
oh ok, yeah that does make a good chunk of sense